librepcb/controlpanel/controlpanel.cpp

```diff
@@ -294,9 +294,11 @@ void ControlPanel::projectEditorClosed() noexcept
 
 void ControlPanel::on_actionAbout_triggered()
 {
-    QMessageBox::about(this, tr("About"), tr(
-        "LibrePCB is a free & open source schematic/layout-editor."
-        "<br><br>"
+    QMessageBox::about(this, tr("About LibrePCB"), tr(
+        "<h1>About LibrePCB</h1>"
+        "<p>LibrePCB is a free & open source schematic/layout-editor.</p>"
+        "<p>Version: " GIT_VERSION "</p>"
+        "<p>Please see <a href='https://librepcb.org/'>librepcb.org</a> for more information.</p>"
```

Comment:
I also have a webspace, that's not the problem. But I don't have the time and nerves to set up the certificate, and even less to set up an auto-updating mechanism...

But as the website contains only static content and thus no sensitive data is transmitted, I would say SSL is not that important (at least until now). There are many other issues which are much more important ;)